### ISSUE TITLE
[BUG] Custom themes don't work

### ISSUE THREAD

**Issue by Sh1d0w** [nylo-core/framework#35]
Hello, thanks for the great framework!

According to the documentation you can create custom themes:

https://nylo.dev/docs/5.x/themes-and-styling#creating-a-theme

This all is good. You can also switch the theme to the custom one:

```
NyTheme.set(context, id: "custom_theme");
```

However the problem occurs when using the colours from the theme:

```
ThemeColor.get(context).primaryContent
```

Because it defaults to either the light or dark theme here https://github.com/nylo-core/support/blob/5.x/lib/helpers/helper.dart#L554 it won't use the theme you set with `NyTheme.set` which makes it pointless.

The way it should work is to use the correct theme set by `NyTheme.set`.

**Comment by agordn52** [comment]
Hi @Sh1d0w,

I'll take a look at my earliest convenience.

**Comment by agordn52** [comment]
This is fixed in [v5.3.5](https://pub.dev/packages/nylo_framework).

Thanks for highlighting the issue 👍

**Comment by Sh1d0w** [comment]
Thanks, works great 🚀
